### DISCUSSION TITLE
I hope to discuss about NFT ERC token

### DISCUSSION THREAD

**Discussion by osadchiidev** [community/community#63034] [Discussions]
### Select Topic Area

Show & Tell

### Body

Please help me for my personal project

**Comment by nightfury-toothless** [comment]
Let me know more details about your NFT project.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
